### ISSUE TITLE
How many epochs are required for training purpose?

### ISSUE THREAD

**Issue by eashan2798** [TensorSpeech/TensorFlowASR#65]
I was trying training conformer model for testing purpose using 1 epoch by running the train_conformer.py file.The loss value is around 400 in one epoch.So what should be the optimal no of epochs for training purpose on Librispeech dataset as right now i am getting WER=100 (weird).Kindly help me out here

**Comment by eashan2798** [comment]
![Screenshot from 2020-12-08 19-07-16](https://user-images.githubusercontent.com/68741106/101523971-56028d80-39af-11eb-9258-526ce93c728d.png)

Here is the screenshot of the output of the test_conformer.py file.Can you tell why this error rate is 100

**Comment by usimarit** [comment]
@eashan2798 The model is not converged, the loss must be around 0-15, you can not test the model if the model hasn't learnt anything. About how many epoch, in my example I placed 20 epochs to reach the WER 6%, I also added the image of the loss logged after each steps, you can use that to decide how many epochs you should use.
